mentors/README.md

```diff
@@ -31,7 +31,7 @@ Each mentor should add a bio to _all_ tracks that they mentor.
 
 ## Add your bio
 Choose the JSON file that starts with the first character of your GitHub username.
-Use the following format.
+Use the following format. 
```

Comment:
```suggestion
Use the following format.
```
Super minor, not sure how the space got inserted, but lets eliminate it so only what you intend to change ends up changed.

Reply:
Oh, probably my fault... Thanks for spotting this one.